### DISCUSSION TITLE
350 working 500 …. Not so much

### DISCUSSION THREAD

**Discussion by jacoats** [GerryDazoo/Slinger#493] [Q&A]
I have two SB’s that been running fine since the config.Ini for two SB’s cane out. A few days ago the slinger server says it can no longer find the 500 in my network and to check the Ini file. 350 still works. Nothing has changed. My router says that my 500 is still on the same lan ip. I can ping it. Could the port have changed ? How can I tell which port the 500 is actually on? Thanks for any help.

**Comment by DVoorhees1957** [comment]
Use a config.ini file that has no slingboxes defined in it... Slinger will report back the slingboxes that it finds

**Reply by GerryDazoo** [reply]
If it doesn't find your slingbox give it a quick power cycle/reset.

**Reply by GerryDazoo** [reply]
If it can't find your slingbox Slinger will probe you LAN looking for one to use and report all that it finds. Are you using the EXE or .py file. 
Check your Slinger output. If your using the .py file make sure you have the netifaces python module installed.

**Reply by jacoats** [reply]
That got it. The port changed to 5207! Thanks a million !
